src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st

```diff
@@ -52,13 +50,8 @@ EpLogBrowserToolbarPresenter >> beRevertMode [
 		action: [ logBrowserPresenter revertAllChangesAndCloseWindow ]
 ]
 
-{ #category : #private }
-EpLogBrowserToolbarPresenter >> filtersAction [
-	logBrowserPresenter filtersSubMenu openWithSpecAt: ActiveHand position
-]
```

Comment:
decided to inline `filtersAction`, looks better and more consistent in the class.